src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateGlobalUsings.targets

```diff
@@ -0,0 +1,65 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.GenerateGlobalUsings.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    ============================================================
+                                     GenerateGlobalUsings
+
+    Generates global using source to intermediate directory
+    ============================================================
+    -->
+  <PropertyGroup>
+    <GeneratedGlobalUsingsFile Condition="'$(GeneratedGlobalUsingsFile)' ==''">$(IntermediateOutputPath)$(MSBuildProjectName).DefaultGlobalUsings$(DefaultLanguageSourceExtension)</GeneratedGlobalUsingsFile>
+    <EnableDefaultGlobalUsings_DotNet Condition="'$(EnableDefaultGlobalUsings_DotNet)' == '' and '$(EnableDefaultGlobalUsings)' == 'true'">true</EnableDefaultGlobalUsings_DotNet>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(EnableDefaultGlobalUsings_DotNet)' == 'true'
+                      and '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
+                      and $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '6.0'))">
+    <DefaultGlobalUsing Include="System" />
```

Comment:
Add System.IO (commonly used).
Add System.Threading.Tasks (need to be used for System.Net.Http, used in Framework template).
Not yet for System.Text.Json (System.Net.Http scenarios)
Not yet for System.Text.Json.Nodes (System.Net.Http scenarios)
Not yet for System.Text.Json.Serialization (System.Net.Http scenarios)
Remove System.Net.Http.Json